### DISCUSSION TITLE
Default Behavior for POST and PUT

### DISCUSSION THREAD

**Discussion by D1g1talEntr0py** [mockapi-io/docs#14] [Ideas]
Shouldn't the POST receive an array of resources, not add a single resource? That is what PUT is for. Which brings me to my other suggestion. Isn't a PUT for adding or updating? When I send a PUT, it doesn't add anything like it does with POST. I don't know. Maybe I'm wrong.

**Comment by nsmirn0v** [comment]
In a REST API, the POST method is typically used to create a new resource on the server, while the PUT method is used to update or replace an existing resource. The specific behavior of these methods may vary depending on the implementation of the API, but this is the general convention followed in most RESTful systems.

**Reply by D1g1talEntr0py** [reply]
I appreciate your reply. I guess my point was why does a POST create a resource while PUT does not?
